### PR TITLE
Hot fix:  AutoRelation" in server creation/update now can be set to false

### PR DESCRIPTION
**Pull request by itakouna**

### CODE REVIEW

File: server_test.go [hunk 1]
```diff
@@ -61,7 +61,7 @@ func TestClient_CreateServer(t *testing.T) {
 		HardwareProfile: "default",
 		AvailablityZone: "",
 		Labels:          []string{"label"},
-		Relations:       &ServerCreateRequestRelations{},
+		//Relations:       &ServerCreateRequestRelations{},
```
Comment (itakouna) [inline server_test.go]: why is it commented?

File: server.go [hunk 2]
```diff
@@ -46,16 +46,16 @@ type ServerRelations struct {
...
+	Name            string   `json:"name"`
+	Memory          int      `json:"memory"`
+	Cores           int      `json:"cores"`
+	LocationUUID    string   `json:"location_uuid"`
+	HardwareProfile string   `json:"hardware_profile,omitempty"`
+	AvailablityZone string   `json:"availability_zone,omitempty"`
+	Labels          []string `json:"labels,omitempty"`
+	Status          string   `json:"status,omitempty"`
+	AutoRecovery    *bool    `json:"auto_recovery,omitempty"`
+	//Relations       *ServerCreateRequestRelations `json:"relations,omitempty"`
```
Comment (itakouna) [inline server.go]: why is it commented?

Comment (nvthongswansea) [reply server.go]: It disappears from the  official API docs and I have tested it. The server returns "Bad request" when we include "relations" field.

Comment (nvthongswansea) [reply server.go]: Just in case it will be back in the future, I just commented this line of code.

Comment (itakouna) [reply server.go]: Then, I would prefer removing all `Relations` and `ServerCreateRequestRelations`.

Comment (nvthongswansea) [reply server.go]: I don't know if it will be back in the future. Is it necessary to remove these lines of code?

Comment (itakouna) [reply server.go]: I don't like keeping codes that are unused. We can add this feature again when API supports it. That why we use git ` version-control system`.

Comment (nvthongswansea) [reply server.go]: Alright then. I will update the code in my branch, I think you will need to pull the update.

**Comment by itakouna** [comment]
@nvthongswansea take a look on the last commit and approve it if it's OK.

**Pull request merged**